e2etest/container/container.go

```diff
@@ -4,17 +4,31 @@ import (
 	"bytes"
 	"context"
 	"fmt"
+	bbn "github.com/babylonlabs-io/babylon/types"
+	"github.com/btcsuite/btcd/btcec/v2"
+	"github.com/cometbft/cometbft/libs/rand"
+	"net"
 	"regexp"
+	"strconv"
 	"testing"
 	"time"
 
 	"github.com/ory/dockertest/v3"
 	"github.com/ory/dockertest/v3/docker"
 	"github.com/stretchr/testify/require"
+	mrand "math/rand/v2"
 )
 
 const (
-	bitcoindContainerName = "bitcoind-test"
+	bitcoindContainerName = "bitcoind"
+	babylondContainerName = "babylond"
+)
+
+var (
+	// jury
+	_, juryPK = btcec.PrivKeyFromBytes(
```

Comment:
```suggestion
	_, covenantPK = btcec.PrivKeyFromBytes(
```

We don't use jury any more